### PR TITLE
CLI, FE and tests adapted for prod mainnet deployment

### PR DESCRIPTION
**Pull request by manolisliolios**

### CODE REVIEW

File: packages/package_info/Move.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@ name = "package_info"
 edition = "2024.beta"
 
 [dependencies]
-Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/mainnet", override=true }
+Sui = { git = "https://github.com/MystenLabs/sui.git", subdir = "crates/sui-framework/packages/sui-framework", rev = "framework/testnet", override=true }
```
Comment (stefan-mysten) [inline packages/package_info/Move.toml:6]: Why is override true here?

File: .github/workflows/rust-ci.yml [hunk 2]
```diff
@@ -26,7 +26,6 @@ jobs:
 
       - name: Build mvr-cli (debug)
         run: |
-          cd mvr-cli
```
Comment (stefan-mysten) [inline .github/workflows/rust-ci.yml:29]: Wouldn't this fail as it needs to build from the `mvr-cli` folder?

Comment (manolisliolios) [reply .github/workflows/rust-ci.yml:29]: I moved the repo to a workspace, so we can use `cargo install --git`! Look at a comment I left on your PR too!

**Review by stefan-mysten — APPROVED** [review]
Thanks @manolisliolios for updating these. Please have a look at the workflow, as it might break the build of the mvr binary.

File: packages/package_info/Move.toml (hunk 1 above)
Comment (manolisliolios) [reply packages/package_info/Move.toml:6]: This is a classic workaround to get around "Sui" versioning story bugs (source verification missmatch)

**Pull request merged**